HyperPlay-Gaming/hyperplay-desktop-client
# [Quests] Post playsession on game close

# Summary

Post play session time to playStreak endpoint on game close

# Testing
1. Log in with HP account
2. Launch Kosium
3. Close kosium
4. check db for updated play session time
